layers/fpm-dev/Dockerfile

```diff
@@ -37,6 +37,9 @@ EXPOSE 8000
 # Add `php/conf.dev.d` to the path where PHP looks for configuration files
 ENV PHP_INI_SCAN_DIR="/opt/bref/etc/php/conf.d:/var/task/php/conf.d:/var/task/php/conf.dev.d"
 
+# Add composer
+COPY --link --from=composer/composer:2-bin /composer /usr/bin/composer
```

Comment:
Can we include the full version, as cache-buster, and to allow explicit upgrading, to ensure a specific composer version is picked up?

Reply:
I'd rather not to keep things simple, this is composer, pretty stable, and in a dev image.